doc/configuration.md

```diff
@@ -43,3 +43,28 @@ Can survive restart. Still needs some other method to act as cache.
 3scale hosts thousands of gateways for its customers and needs a reasonable way to share resources between them. Multi-tenant deployment of this proxy is the preferred way.
 
 TODO: figure out how to store/load the configuration in multi-tenant way
+
+
+## How config is loaded:
+
+On APICast there are three main components regarding configuration, the
+components are the following ones:
```

Comment:
**Suggestion:** The following are the three main components in an APIcast configuration: